from arango_internals.exceptions import ArangoError


class JobResultError(ArangoError):
    """Job response does not yet exist."""
